geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/GetCommand.java

```diff
@@ -61,16 +60,6 @@ public Result get(
     cache.getSecurityService().authorize(Resource.DATA, Operation.READ, regionPath, key);
     DataCommandResult dataResult;
 
-    if (StringUtils.isEmpty(regionPath)) {
-      return makePresentationResult(DataCommandResult.createGetResult(key, null, null,
-          CliStrings.GET__MSG__REGIONNAME_EMPTY, false));
-    }
-
-    if (StringUtils.isEmpty(key)) {
-      return makePresentationResult(DataCommandResult.createGetResult(key, null, null,
-          CliStrings.GET__MSG__KEY_EMPTY, false));
-    }
-
```

Comment:
I'm just curious where this validation happens now.

Reply:
the parameter is marked as "mandatory", so the parser should have caught the error way before this.